packages/stores/src/queries-external/token-pair-historical-chart/index.ts

```diff
@@ -0,0 +1,92 @@
+import { KVStore } from "@keplr-wallet/common";
+import { HasMapStore } from "@keplr-wallet/stores";
+import { Dec } from "@keplr-wallet/unit";
+import { computed, makeObservable } from "mobx";
+import { IPriceStore } from "src/price";
+
+import { IMPERATOR_HISTORICAL_DATA_BASEURL } from "..";
+import { ObservableQueryExternalBase } from "../base";
+import { TokenPairHistoricalPrice } from "./types";
+
+const AvailableRangeValues = ["7d", "1mo", "1y"] as const;
```

Comment:
Consider a union type like `"7d" | "1mo" | "1y"`, and make it exportable from types.ts.